integration-tests/azure-app-configuration/README.md

```diff
@@ -12,6 +12,44 @@ To successfully run this sample, you need:
 * Azure CLI and Azure subscription if the specific Azure services are required
 * Docker if you want to build the app as a native executable
 
+You also need to make sure the right version of dependencies are installed.
+
+### Use development iteration version
+
+By default, the sample depends on the development iteration version, which is `999-SNAPSHOT`. To install the development
+iteration version, you need to build it locally.
+
+```
+# Switch to the root directory of Quarkus Azure services extensions.
+# For example, if you are in the directory of quarkus-azure-services/integration-tests/azure-app-configuration
+cd ../..
+
+# Install all Quarkus Azure services extensions locally.
+mvn clean install -DskipTests
+
+# Switch back to the directory of integration-tests/azure-app-configuration
+cd integration-tests/azure-app-configuration
+```
+
+### Use release version
+
+If you want to use the release version, you need to update the version of dependencies in the `pom.xml` file.
+
+First, you need to find out the latest release version of the Quarkus Azure services extensions
+from [releases](https://github.com/quarkiverse/quarkus-azure-services/releases), for example, `1.0.0`.
+
+Then, update the version of dependencies in the `pom.xml` file, for example:
+
+```xml
+
+<parent>
+    <groupId>io.quarkiverse.azureservices</groupId>
+    <artifactId>quarkus-azure-integration-test</artifactId>
+    <version>1.0.0</version>
+    <!--        <relativePath>../pom.xml</relativePath>-->
```

Comment:
![image](https://github.com/quarkiverse/quarkus-azure-services/assets/4465723/c2ca5190-ab7b-4c26-b4e7-201c5a07ac8a)
can we remove this line here?

Reply:
Due to the recent commit and local testing, `relativePath` needs to be kept to eliminate the warning so that it will resolve the parent POM from the repositories for a specified version. See [this](https://stackoverflow.com/questions/6003831/parent-relativepath-points-at-my-com-mycompanymyproject-instead-of-org-apache) for more information.